internal/ycsb/ycsb.go

```diff
@@ -63,11 +84,22 @@ func Run(ctx context.Context, dir string, args []string) (*config.TestResults, e
 
 	cmd = exec.CommandContext(ctx, bin, cliArgs...)
 	cmd.Dir = dir
-	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
 
+	pipe, err := cmd.StdoutPipe()
+	if err != nil {
+		return nil, err
+	}
+
+	defer pipe.Close()
+
 	log.Printf("Running %s", strings.Join(cmd.Args, " "))
 
+	var buf strings.Builder
+	mw := io.MultiWriter(os.Stdout, &buf)
```

Comment:
@AlekSi I couldn't make it working with `TeeReader`, but this approach seems to work. Please feel free to push the code that fixes it if you see how it could be done.

Reply:
That's because you did not remove `strings.Builder` and conversion to string.
Make `parseMeasurements` accept `io.Reader`, not `string`

Reply:
I did it around this commit - https://github.com/FerretDB/dance/pull/944/commits/556f5a8fe0d2223f99c93929f1766c4840f60516, it didn't work.

Reply:
556f5a8fe0d2223f99c93929f1766c4840f60516 and 60557894c1ec2e25fa1811903b6ed339c8b99753 don't call `Start` method at all, and 1876e427db253468fdb02591a47bbef1c6d68cbc calls `Wait` right after `Start`.

[Documentation](https://pkg.go.dev/os/exec#Cmd.StdoutPipe) says:

> It is thus incorrect to call Wait before all reads from the pipe have completed.
> […]
> See the example for idiomatic usage.

Calling `Wait` before reads completed (or even started is what the last PR did.
The [example](https://pkg.go.dev/os/exec#example-Cmd.StdoutPipe) is really close to our situation.

Reply:
2b24b6d602dffdf265343c4bb4f73a11dddc479b